src/common/logging.ts

```diff
@@ -29,11 +31,28 @@ const logging =
         stderrLevels: ['error'],
         consoleWarnLevels: ['warn', 'debug', 'info'],
       }),
-      new Sentry({
-        dsn: 'https://34760911653f40a18ba07c39a6a37002@sentry.io/1488146',
-        level: 'info',
-      }),
     ],
   });
 
+if (envVariables.logglyToken !== null &&
```

Comment:
`if (!envVariables.logglyToken) { ... }` is the same as making sure it's not null, undefined, or an empty string.